.github/workflows/ci.yaml

```diff
@@ -9,10 +9,22 @@ on:
   workflow_dispatch: # allows you to trigger manually
 
 jobs:
+  pre-job:
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          concurrent_skipping: "same_content"
+          skip_after_successful_duplicate: "true"
+          do_not_skip: '["pull_request", "workflow_dispatch"]'
```

Comment:
Incidentally, this action can be used to automatically skip CI runs for documentation related PRs (without needing to use the `[skip-ci]` in the commit message). 


```yaml
with:
	paths_ignore: '["**/doc/**"]'
```

Should we enable this?

Reply:
> Should we enable this?

:+1:, it's really easy to forget adding that flag

Reply:
Great idea!